src/openapi.yaml

```diff
@@ -335,6 +339,8 @@ paths:
     $ref: './resources/DatabaseRestart.yaml'
   /database/{databaseId}/stop:
     $ref: './resources/DatabaseStop.yaml'
+  /database/{databaseId}/cancelDeployment:
```

Comment:
Cancel is only available at the environment level. It will not be possible to cancel a single service inside an env

Reply:
Understood, I'll review frontend workflow consequently.

Reply:
I removed application endpoint "Cancel deployment"